src/backend/internal/errors/errors.go

```diff
@@ -0,0 +1,17 @@
+package errors
+
+import (
+	"github.com/gin-gonic/gin"
+	log "github.com/sirupsen/logrus"
+)
+
+// ValidationErrors Users input validation errros
+var ValidationErrors = []string{}
+
+//HandleErr //generic error handler, logs error and Os.Exit(1)
+func HandleErr(c *gin.Context, err error) error {
```

Comment:
@adefemi171  How do you plan to use the Err handler?

Reply:
Was going to use it for db errors, but will remove it from the code